lib/game.rb

```diff
@@ -1,22 +1,32 @@
 class Game
   attr_reader :current_player
 
-  def initialize(player1, player2)
+  def initialize(board, display, player1, player2)
+    @board = board
+    @display = display
     @player1 = player1
     @player2 = player2
     @current_player = player1
   end
 
-  def prompt_make_move
+  def prompt_for_move
     move = $stdin.gets.chomp
     move.to_i
   end
 
-  def toggle_player
+  def toggle_current_player
     if @current_player == @player1
       @current_player = @player2
     else
       @current_player = @player1
     end
   end
+
+  def make_move
+    @display.display_board(@board.board)
+    @display.ask_for_move
+    move = prompt_for_move
```

Comment:
What is the difference between asking for a move and prompting for a move?

Reply:
The difference is ask_for_move displays "Choose a move between 1-9" and prompt_for_move requests input. 
Maybe the prompt should be in ask_for_move, and then prompt_for_move will be redundant. What do you think?